### PR TITLE
Boshi Logo

### PR DESCRIPTION
**Pull request by BoshiBase**
Please review the following token assets:

## 📑 Description

-   Token Name: Boshi
-   Project Website: https://www.boshibase.com/
-   Explorer URI: https://basescan.org/address/0x33ad778e6c76237d843c52d7cafc972bb7cf8729

---

## ✅ Checks

-   [x] I created a new folder with the token address, **all in lowercase**
-   [x] I added the token's logo as a `32x32` PNG file, named `logo-32.png`
-   [x] I added the token's logo as a `128x128` PNG file, named `logo-128.png`
-   [x] I added the token's logo as a SVG file, named `logo.svg`
-   [x] My SVG logo is a proper SVG and not some base64 stuff
-   [x] My documentation/website clearly display the token address somewhere

### CODE REVIEW

File: logo.svg
Comment (abmisx0) [inline logo.svg:1]: please remove this file from the PR

**Review by BoshiBase — COMMENTED** [review]
Updated SVG logo

**Review by BoshiBase — COMMENTED** [review]
Updated SVG

**Review by BoshiBase — COMMENTED** [review]
Updated

File: logo.svg
Comment (BoshiBase) [reply logo.svg:1]: Done

**Review by abmisx0 — APPROVED** [review]

**Pull request merged**